packages/cloud_functions/ios/Classes/CloudFunctionsPlugin.m

```diff
@@ -35,36 +35,46 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
   if ([@"CloudFunctions#call" isEqualToString:call.method]) {
     NSString *functionName = call.arguments[@"functionName"];
     NSObject *parameters = call.arguments[@"parameters"];
-    [[FIRFunctions functions]
-        callFunction:functionName
-          withObject:parameters
-          completion:^(FIRHTTPSCallableResult *callableResult, NSError *error) {
-            if (error) {
-              FlutterError *flutterError;
-              if (error.domain == FIRFunctionsErrorDomain) {
-                NSDictionary *details = [NSMutableDictionary dictionary];
-                [details setValue:[self mapFunctionsErrorCodes:error.code] forKey:@"code"];
-                if (error.localizedDescription != nil) {
-                  [details setValue:error.localizedDescription forKey:@"message"];
-                }
-                if (error.userInfo[FIRFunctionsErrorDetailsKey] != nil) {
-                  [details setValue:error.userInfo[FIRFunctionsErrorDetailsKey] forKey:@"details"];
-                }
+    NSString *appName = call.arguments[@"app"];
+    NSString *region = call.arguments[@"region"];
+    FIRApp *app = [FIRApp appNamed:appName];
+    FIRFunctions *functions;
+    if (region != nil) {
+      functions = [FIRFunctions functionsForApp:app region:region];
+    } else {
+      functions = [FIRFunctions functionsForApp:app];
+    }
+    [functions callFunction:functionName
+                 withObject:parameters
+                 completion:^(FIRHTTPSCallableResult *callableResult, NSError *error) {
+                   if (error) {
+                     FlutterError *flutterError;
+                     if (error.domain == FIRFunctionsErrorDomain) {
+                       NSDictionary *details = [NSMutableDictionary dictionary];
+                       [details setValue:[self mapFunctionsErrorCodes:error.code] forKey:@"code"];
```

Comment:
I didn't write this code, it was already there. `setObject` does seem more appropriate, but I think that should be a different PR. Filed https://github.com/flutter/flutter/issues/28271

Reply:
I am ok with having this since setValue:forKey seems to do the same thing as setObject:forKey for NSMutableDictionary. 
https://developer.apple.com/documentation/foundation/nsmutabledictionary/1416335-setvalue?preferredLanguage=occ